viewer/src/data/model/PointCloud.ts

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import { File3dFormat } from './File3dFormat';
+import { PromiseCallbacks } from './PromiseCallbacks';
+import { CdfSource } from './CdfSource';
+import { ExternalSource } from './ExternalSource';
+import { PotreeGroupWrapper } from '../../views/threejs/pointcloud/PotreeGroupWrapper';
+import { PotreeNodeWrapper } from '../../views/threejs/pointcloud/PotreeNodeWrapper';
+
+export interface PointCloud {
+  format: File3dFormat.EptPointCloud;
+  source: CdfSource | ExternalSource;
+  callbacks: PromiseCallbacks<[PotreeGroupWrapper, PotreeNodeWrapper]>;
```

Comment:
Why does this thing in `data/model` depend on something in `views/threejs`?

Reply:
Because of the current solution of callbacks which we want to remove at some point, but for current implementation we are stuck with the callback (this is something I really want to change as soon as we have time, I don't foresee it being a big job)